src/main/resources/application-local.yml

```diff
@@ -24,3 +25,15 @@ aws:
         secret-key: 12345
         bucket-kms-key:
       bucket-name: untrusted-bucket
+
+  case:
```

Comment:
This shouldn't be required - you have also committed the user and team here.

Reply:
Fixed

Reply:
This still shouldn't have the `group`, `team`, `user`. here